aws/idp/waf.tf

```diff
@@ -1,4 +1,7 @@
-# TODO: add AWS Common Ruleset once service has been stood up
+locals {
+  excluded_common_rules = ["EC2MetaDataSSRF_Body"]
```

Comment:
should we add an explanation why we are excluding?

Reply:
good idea - I'll get a new PR in with a comment to explain